Assets/Scripts/UI/FloatingInfoPanel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using TMPro;
+
+public class FloatingInfoPanel : MonoBehaviour, IPointerDownHandler, IPointerExitHandler, IPointerEnterHandler
```

Comment:
So again, I don't know, but what if this panel itself had a method for setting the current info object?

Reply:
Yeah, that might end up being better.  There's some redundancy in having this info set externally.  There's likely going to be another round of updates to these values since the Sun and Moon values are looking to be incorrect and require extra manipulation.  I'm going to try to do some of that refactor then.  Thanks for the input!